basic_counter/assets/css/app.css

```diff
@@ -34,3 +34,11 @@
       url("/fonts/Poppins-SemiBold.woff") format("woff");
   }
 }
+
+a {
```

Comment:
I'd rather use Tailwind's Prose block for consistency, but I'll modify it to match your expectations

Reply:
The prose block imposes a narrow max width. I adjusted the layout to remove it so the cards and the text were same container and the main max-width defers to the layout in the website where this will be included.

Ie: I deliberately removed the prose class.

Reply:
Yeah, that's also easy to fix with [`max-w-none`](https://tailwindcss.com/docs/typography-plugin#overriding-max-width) class on the `prose` element. I think a prose element is easier to manage when writing a bunch of text, so that you don't need to worry about inner styling/classes

Reply:
Makes sense, can I leave the tweaks to you?

Reply:
Yup, on it, as well as adding a dark theme

Reply:
🕶️